src/libmongoc/doc/ref/packages.rst

```diff
@@ -0,0 +1,213 @@
+##############################
+Package Installation Reference
+##############################
+
+.. _EPEL: https://docs.fedoraproject.org/en-US/epel/
+.. _conan: https://conan.io/
+.. _vcpkg: https://vcpkg.io/
+
+|libbson| and |libmongoc| are available from several package management tools
+on a variety of systems.
+
+.. important::
+
+  The third-party packages detailed here are not directly controlled via the
+  |mongo-c-driver| maintainers, and the information found here may be incomplete
+  or out-of-date.
+
+
+Package Names and Availability
+******************************
+
+This table details the names and usage notes of such packages.
+
+.. seealso::
+
+  For a step-by-step tutorial on installing packages, refer to
+  :doc:`/learn/get/installing`.
+
+
+.. list-table::
+  :name: Available Packages
+  :header-rows: 1
+  :align: left
+  :widths: auto
+
+  - - Packaging Tool
+    - Platform(s)
+    - |libbson| package(s)
+    - |libmongoc| package(s)
+    - Notes
+
+  - - :index:`APT <!pair: APT; package names>` (``apt``/``apt-get``)
+    - Debian-based Linux distributions (:index:`Debian`, :index:`Ubuntu`, Linux
+      Mint, etc.)
+    - ``libbson-1.0-0``, ``libbson-dev``,
+      ``libbson-doc``
+    - ``libmongoc-1.0-0``, ``libmongoc-dev``,
+      ``libmongoc-doc``
+    - .. empty cell
+
+  - - :index:`Yum <!pair: Yum; package names>`/:index:`DNF <!pair: DNF; package names>`
+    - RHEL-based systems (RHEL, :index:`Fedora`, :index:`CentOS`,
+      :index:`Rocky Linux`, :index:`AlmaLinux`)
+    - ``libbson``, ``libbson-devel``
+    - ``mongo-c-driver-libs``, ``mongo-c-driver-devel``
+    - *Except on Fedora* the EPEL_ repositories must be enabled (i.e. install
+      the ``epel-release`` package first)
+
+  - - :index:`APK <!pair: APK; package names>`
+    - :index:`Alpine Linux <!pair: Alpine Linux; package names>`
+    - ``libbson``, ``libbson-dev``, ``libbson-static``
+    - ``libmongoc``, ``libmongoc-dev``, ``libmongoc-static``
+    - .. empty cell
+
+  - - :index:`pacman <!pair: package names; pacman>`
+    - :index:`Arch Linux <!pair: package names; Arch Linux>`
+    - ``mongo-c-driver``
+    - ``mongo-c-driver``
+    - A single package provides both runtime and development support for both
+      |libbson| and |libmongoc|
+
+  - - :index:`Homebrew <!pair: Homebrew; package names>`
+    - :index:`macOS <!pair: macOS; package names>`
+    - |ditto|
```

Comment:
Should these `|ditto|`s be changed to `.. empty cell`? Or do they not mean the same thing?

If the latter, rather than `‘’` as the replacement text, suggest "_Same as cell above_" instead for better clarity.